cmd/kubebuilder/initproject/init.go

```diff
@@ -91,6 +91,7 @@ func runInitRepo(cmd *cobra.Command, args []string) {
 	doInject(cr)
 	doArgs(cr, controllerOnly)
 	//os.MkdirAll("bin", 0700)
+	doImports(cr)
```

Comment:
We might have to invoke it in `vendor update` workflow as well. See my comment below.